tests/EntityManager.mongo.test.ts

```diff
@@ -846,6 +846,12 @@ describe('EntityManagerMongo', () => {
     book = (await orm.em.findOne(Book, book._id))!;
     expect(book.tags.count()).toBe(2);
 
+    // slice
+    expect(book.tags.slice()).toEqual(items);
+    expect(book.tags.slice(0, 2)).toEqual(items);
+    expect(book.tags.slice(1)).toEqual(items[1]);
```

Comment:
as already said, this is wrong, you cant compare array with the item itself, should be something like following instead:

```suggestion
    expect(book.tags.slice(1)).toEqual([items[1], items[2]]);
```